### DISCUSSION TITLE
Attempted to serialize scalar '2021-03-03 00:00:00 +00:00' with incompatible type 'String' for field date.

### DISCUSSION THREAD

**Discussion by chrockny** [prisma/prisma#7703] [Q&A] [answered]
(node:4925) UnhandledPromiseRejectionWarning: Error: 
Invalid `prisma.users.findMany()` invocation:

  Attempted to serialize scalar '2021-03-03 00:00:00 +00:00' with incompatible type 'String' for field date.
    at cb (/mnt/c/Users/51935/Documents/CIBERGRAM/node_modules/@prisma/client/runtime/index.js:33820:17)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:4925) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:4925) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

I can't find a solution, help

**Comment by ryands17** [comment]
@chrockny 👋 

Could you share the `findMany` and what you're sending as arguments?

**Reply by huayemao** [reply]
similar problem, I did a `findFirst({ orderBy: { date: 'desc' } })`

**Reply by huayemao** [reply]
> similar problem, I did a `findFirst({ orderBy: { date: 'desc' } })`

and it only happens on vercel serverless function execution in my case, works fine in my local , seems to be related to platform.

**Reply by huayemao** [reply, marked as answer]
> > similar problem, I did a `findFirst({ orderBy: { date: 'desc' } })`
> 
> and it only happens on vercel serverless function execution in my case, works fine in my local , seems to be related to platform.

Oh, I figured out it's the cache, I changed the schema locally, but didn't rebuild on the cloud

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
